terrible hack to fix cross builds
overwrite HOST_* flags to get rid of -MF

note: this patch was used from firefox-77 until 80, dropped with the update to 81

--- a/third_party/rust/glslopt/build.rs
+++ b/third_party/rust/glslopt/build.rs
@@ -28,6 +28,8 @@
     env::remove_var(format!("CXXFLAGS_{}", &target));
     env::remove_var(format!("CFLAGS_{}", target.replace("-", "_")));
     env::remove_var(format!("CXXFLAGS_{}", target.replace("-", "_")));
+    env::set_var("HOST_CFLAGS", "-O2");
+    env::set_var("HOST_CXXFLAGS", "-O2");

     configure(&mut cc::Build::new())
         .warnings(false)
